setup.sh

```diff
@@ -1,21 +1,47 @@
 #!/bin/bash
 #
-# Sets up parameters for use with other scripts.  Should be run once.
+# Sets up parameters for use with other scripts.  Removes an instance of param.sh if it exists. 
 # Sample usage: bash setup.sh
-
+# Can be run for any number of times
+rm -r params.sh
 echo "First, choose the resource identifier that specifies your cluster resoure. We
 will set up this name in your ssh configuration, and use it to reference the resource (sherlock)."
 echo
 read -p "Resource identifier (default: sherlock) > "  RESOURCE
 RESOURCE=${RESOURCE:-sherlock}
 
+if [[ "${RESOURCE}" == "sherlock" ]]
+then
+   SHERLOCK=true
+else 
+   SHERLOCK=false
+fi
+
+echo "Next, please supply the domain name of the resource. If you are using farmshare or sherlock, leave as default."
+echo
+read -p "Domain Name for Resource (default: rice.stanford.edu for farmshare) > " DOMAINNAME
+DOMAINNAME=${DOMAINNAME:-rice.stanford.edu}
+
+echo "Next, please supply the prefix of the compute nodes that are used in your cluster resource. We will use this to check for assignment of 
+compute node when we submit the sbatch script. If you are using sherlock or farmshare (without gpu capability), then prefixes are set for you."
+echo
+read -p "Compute Node Prefix identifier (default: sh for sherlock) > "  MACHINEPREFIX 
```

Comment:
This also doesn't make sense - if they have already provided sherlock / farmshare we can make the setting quietly without giving them a message. We only need the message if they haven't chosen farmshare or sherlock because we don't know the machine prefix.